keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2180,11 +2066,39 @@ message NodePrepareResourcesResponse {
     // will be called again for those that are missing.
     map<string, NodePrepareResourceResponse> claims = 1;
 }
+
+message NodePrepareResourceResponse {
+    // These are the additional devices that kubelet must
+    // make available via the container runtime. A claim
+    // may have zero or more requests and each request
+    // may have zero or more devices.
+    repeated Device cdi_devices = 1 [(gogoproto.customname) = "CDIDevices"];
+    // If non-empty, preparing the ResourceClaim failed.
+    // cdi_devices is ignored in that case.
+    string error = 2;
+}
```

Comment:
Was this not in the KEP previously?

Reply:
It was, but not in a way that allowed kubelet to associate the CDI IDs with claim requests.

This revised API enables that by adding the request name.

Reply:
I just didn't see a `-` for a similar struct anywhere in the diff.

Reply:
We had `NodePrepareResourcesResponse` before, but it contained `Claims`. Now we need this additional struct.

Reply:
Or something like this - I haven't actually checked the diff...